CONTRIBUTING.md

```diff
@@ -1,3 +1,15 @@
 # How to Contribute to KubeArmor?
 
-See [Contribution Guide](contribution/contribution_guide.md) and [Development Guide](contribution/development_guide.md) for detailed information.
+1. Pick [issue](https://github.com/kubearmor/KubeArmor/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) to work on. For first-time contributors, best would be to pick issue [tagged good-first-issue](https://github.com/kubearmor/KubeArmor/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22).
+2. If you feel adventurous and would like to take a challenging issue, we would love to have you contribute on issues [tagged help-wanted](https://github.com/kubearmor/KubeArmor/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22).
```

Comment:
We should update the links to be organization wide maybe? This way we can include issues from kubearmor-client and other related repositories...

1. Issues - https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Akubearmor
2. Good First Issues - https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Akubearmor+label%3A%22good+first+issue%22
3. Help Wanted Issues - https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Akubearmor+label%3A%22help+wanted%22